components/atoms/typography.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components'
+import { Green050, Grey010 } from './colors'
+
+export const PageTitle = styled.h1`
+        font-family: 'Figtree', sans-serif;
+        font-style: normal;
+        font-weight: 600;
+        font-size: 64px;
+        line-height: 120%;
+        color: ${Green050};
+    `
+
+export const H1 = styled.h1`
+    font-family: 'Gentium Book Basic', serif;
```

Comment:
I don't see a lot of duplication here, but in the future it would have some advantages to export some of these stylings in a const, so you would have something like

```
const fontA = `
  font-family: 'a';
  font-weight: 123;
  line-height: 123%;
`
```

const compA = styled.div`
  ${fontA};
  
  ...
`